README.md

```diff
@@ -26,43 +26,14 @@ A simple example on how to connect to a platform and use it execute a pulse sequ
 
 ```python
 from qibolab import create_platform, ExecutionParameters
```

Comment:
If #1052 is approved
```suggestion
from qibolab import create_platform
```

Reply:
ee0b9d07478a45dd9a1ab159bb8d1f82f4d431ec